app/views/candidate_interface/other_qualifications/details/_form.html.erb

```diff
@@ -20,5 +20,6 @@
                            width: ['A level', 'AS level', 'GCSE'].include?(@form.qualification_type) ? 4 : 10,
                            hint: @form.grade_hint %>
     <%= tag.div(id: 'grade-autosuggest-data', data: { source: grades }) if grades %>
-    <%= f.govuk_text_field :award_year, label: { text: t('application_form.other_qualification.award_year.label'), size: 'm' }, hint: { text: t('application_form.other_qualification.award_year.hint_text') }, inputmode: 'numeric', width: 4 %>
+    <%= f.govuk_text_field :award_year,
+                           label: { text: t('application_form.other_qualification.award_year.label'), size: 'm' }, hint: { text: t('application_form.other_qualification.award_year.hint_text') }, inputmode: 'numeric', width: 4, maxlength: 4 %>
```

Comment:
Another inputmode: numeric -- I know this was there before you picked it up, but since we have checked the advice, we might as well get it right now.